sites/all/modules/ding_unilogin/ding_unilogin.install

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the ding_unilogin module.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function ding_unilogin_install() {
+  db_update('system')
+    ->fields(array(
+      'weight' => 100,
```

Comment:
:+1: The important part here IMO is to document which other alter function we had to run after for the module to work.